### ISSUE TITLE
pass traceback object to allow catching all exceptions using sys.excepthook

### ISSUE THREAD

**Issue by noxqs** [ploomber/debuglater#5]
If you add a parameter tb in function run in pydump.py:
```
def run(filename, echo=True, tb=None):
    out = Path(filename).with_suffix('.dump')
    if echo:
        print(Fore.RED + f'Exception caught, writing {out}\n')
    save_dump(out, tb)
    if echo:
        print(f'To debug, run:\n  dltr {out}')
        print(Style.RESET_ALL)
```

then you can catching all exceptions like:

```
import sys

def my_except_hook(exctype, value, traceback):
    if exctype == KeyboardInterrupt:
        sys.exit("Ctrl-C pressed")
    else:
        import debuglater
        debuglater.run(__file__, tb=traceback)
        sys.__excepthook__(exctype, value, traceback)

sys.excepthook = my_except_hook


if __name__ == '__main__':
    x = 1
    y = 0
    x / y
```

**Comment by edublancas** [comment]
this makes sense, it'd be a lot simpler than cathing the exceptions!

do you have some free time? PRs are welcome :)

**Comment by noxqs** [comment]
Don't have a lot of time but feel free to incorporate it in your project.
I will be testing it out on the side, maybe email the dumps or store them on cloud for easier remote debugging.

**Comment by edublancas** [comment]
sure, user testing is highly appreciated! I'll keep you posted on progress

**Comment by edublancas** [comment]
this is available in 1.4.2 - https://github.com/ploomber/debuglater#usage

```python
import sys
import debuglater

sys.excepthook = debuglater.excepthook_factory(__file__)
```

**Comment by noxqs** [comment]
Awesome ! Thanks
